scripts/postgres/start-dev

```diff
@@ -7,6 +7,10 @@ rm -rf ./.data
 # Initialize the database cluster
 initdb -D ./.data --no-locale
 
+# allow docker connection to the db.
+sed -i "/listen_addresses/c\listen_addresses = \'*\'" ./.data/postgresql.conf
```

Comment:
Okay

Reply:
Nvm, getting invalid command code .

Reply:
I correct Mac specific errors. I should work now on mac.